components/PersonalFinance/PersonalFinanceLayout.js

```diff
@@ -28,176 +25,9 @@ export function PersonalFinanceLayout() {
   const [hideHomePageItems, setHideHomePageItems] = useState(false);
   const [hideTransactionPageItems, setHideTransactionPageItems] = useState(false);
   const [hideIncomeExpensePageItems, setHideIncomeExpensePageItems] = useState(false);
-  //Payments percentage values of categories for pie chart
-  const [expenseData, setExpenseData] = useState([]);
-  //Monthly income values for bar chart
-  const [incomeData, setIncomeData] = useState([]);
-  //Sum of monthly average expenses
-  const [expenseMonthlyAvgData, setExpenseMonthlyAvgData] = useState(0);
-  //Sum of monthly average incomes
-  const [incomeMonthlyAvgData, setIncomeMonthlyAvgData] = useState(0);
-  //Monthly sum of payments in categories
-  const [expenseMonthlyData, setExpenseMonthlyData] = useState([]);
-  //All payments by categories
-  const [paymentsData, setPaymentsData] = useState([]);
-  //Expense values grouped by day
-  const [expensesByDate, setExpensesByDate] = useState([]);
-  //Income values grouped by day
-  const [incomesByDate, setIncomesByDate] = useState([]);
-  //If refresh connection returns error
-  const [refreshConnectionError, setRefreshConnectionError] = useState(false);
-  const [expenseLoading, setExpenseLoading] = useState(true);
-  const [incomeLoading, setIncomeLoading] = useState(true);
-
-  function setIncomeExpenseData() {
-    const userId = sessionStorage.getItem("userId");
-
-    //Before creating income & expense summary, creating or refreshing the relevant connections is required
-    axios
```

Comment:
It is really great work @MahmutKasimoglu Only thing I will raise that add create income & expense requests back to Home page.